CRD-DTR/HomeOxygenTherapy/R4/resources/Questionnaire-R4-HomeOxygenTherapy.json

```diff
@@ -371,13 +387,18 @@
           "text": "Current Order Description",
           "type": "string",
           "readOnly": true,
```

Comment:
This field is readOnly so the user can NOT change it. Therefore, putting constraints here doesn't make sense.  Need to find another field, e.g. free text field to put the constraint.